### PR TITLE
feat: create docusaurus

### PR DESCRIPTION
**Pull request by Nebulis**

### CODE REVIEW

**Comment by simboonlong** [comment]
### General
- Alls links regarding to OA should open in new tab? otherwise i thought i was still in OC documentation and i got lost trying to navigate back.
- is it a prerequisite to know how blockchain and etherum address works b4 reading the docs? if it is, maybe we should mention it right at the start, pointing users to some basic blockchain + etherum materials as there are mentions of terms like "registry", "mainnet" etc.

### Design
- keep a current version number somewhere persistent on navbar? (eg)
react-version

File: docs/schema.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+> Don't forget to replace `$template.url`, `issuers[0].documentStore` and `issuers[0].documentStore.identityProof.location` as explained in OpenAttestation documentation
+
+Run the cli using the schema option:
+
+```json
+❯  ./open-attestation-linux wrap unwrapped wrapped --schema https://schema.opencerts.io/transcripts/2.0
+✔  success   Batch Document Root: 0xc5507674eb34c36343d0da6a79a76c7967c5f3b1f7642c74ea822e7cff1b8a69
+```
+
```
Comment (simboonlong) [inline docs/schema.md]: maybe something like `open-attestation-linux wrap` accepts 2 arguments <input folder> <output folder>, so a typical operation looks like this `./open-attestation-linux wrap input output`

Comment (Nebulis) [reply docs/schema.md]: Added a note

**Comment by Nebulis** [comment]
- added more details in getting started section
- removed link to v1 doc in the header
- added current version in the header (v2)
- added a new section for legacy links
- added a section in the cli to explain it a bit
- added few words in verifier to get more details from OA doc
- external links

**Pull request merged**